lists/global.csv

```diff
@@ -1114,3 +1114,8 @@ http://zoomshare.com/,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,
 https://www.opentech.fund/,HUMR,Human Rights Issues,2017-09-26,OONI,
 https://www.womenonwaves.org/,HUMR,Human Rights Issues,2017-10-26,OONI,
 https://www.womenonweb.org/,HUMR,Human Rights Issues,2017-10-26,OONI,
+https://www.rferl.org/,NEWS,News Media,2017-09-17,igorv,
+https://www.radiosvoboda.org/,NEWS,News Media,2017-09-17,igorv,
+https://vk.com/,GRP,Social Networking,2018-02-23,igorv,
+https://ok.ru/,GRP,Social Networking,2018-02-23,igorv,
```

Comment:
I think all the sites above should be in the russian and/or ukraine list. They contain content that is written in cyrillic.